### ISSUE TITLE
Shutting down IntelliSense server on MacOs

### ISSUE THREAD

**Issue by noppej** [microsoft/vscode-cpptools#3225]
**Type: LanguageService**

**Describe the bug**
- OS and Version: MacOS Mojave 10.14.3
- VS Code Version: 1.31.1
- C/C++ Extension Version: 0.22.0 (Insiders). I only tried Insiders because of this problem in the default release
- Other extensions you installed (and if the issue persists after disabling them): Cortex-Debug, FreeMarker, Python. The problem persists even when I disable these.
- A clear and concise description of what the bug is.
I get a large number of "Problems" flagged by IntelliSense. My c_cpp_properties file config works well and even allows me to build and debug, so I don't think the problem lies with my code/config (although experience suggests anything is possible ... lol)

When I turn on logging ("Debug" level) for the C extension, I get the error "Shutting down IntelliSense server:" when I try to open .c files in my project.

**To Reproduce**

1. Go to ' Command Palette, and  >Reload Window'
2. Click on 'any .c file'
4. See error: Many "Problems" and the error message of this issue in the "Output" for the extension

**Expected behavior**

**Screenshots**

![screen shot 2019-02-28 at 4 05 19 pm](https://user-images.githubusercontent.com/5413046/53598586-2506ef00-3b73-11e9-8309-0d6fbf822b09.png)
![screen shot 2019-02-28 at 4 06 09 pm](https://user-images.githubusercontent.com/5413046/53598534-07398a00-3b73-11e9-9a66-cdcd9fde975e.png)

**Additional context**

**Comment by bobbrow** [comment]
This is not an error.  We periodically shut down servers when too many are open.  We don't want to use **all** of your computer's memory. 😉

The logs show that a server for sstpCommon.c was opened and main.c was closed so you must be hovering around the memory usage limit we set for ourselves.  The problems listed are actual problems with the new server (sstpCommon.c).  It has a bunch of undefined symbols.  Perhaps a `#define` is missing and causing declarations in your headers to be skipped.
